### PR TITLE
Update vn.m3u

### PR DESCRIPTION
**Pull request by taodicakhia**
Rewrite all vn.m3u

### CODE REVIEW

**Review by freearhey — CHANGES_REQUESTED** [review]
Please put the working links back in the playlist.

**Comment by taodicakhia** [comment]
> Please put the working links back in the playlist.

Some working links in previous time are already in here. I rewrite vn.m3u to rearrange channels in Vietnam

**Comment by freearhey** [comment]
All of the `livefta.malimarcdn.com` links work, but I don't see them in your `vn.m3u` file or anywhere else. If you decide to transfer these links to other files, they must also be included in this pull request.

File: channels/vn.m3u [hunk 1]
```diff
@@ -1,130 +1,1075 @@
...
+https://vips-livecdn.fptplay.net/sda/gialai_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="GiaLai.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Gia Lai (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sda/gialai_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="GiaLai.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Gia Lai (486p)
 http://113.161.25.3:8134/hls/gialaitv/gialaitv.m3u8
-#EXTINF:-1 tvg-id="GoodIdeaTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Good Idea TV (720p)
-https://livefta.malimarcdn.com/ftaedge00/gooditv.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="HBTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://i.imgur.com/BnFWgS7.png" group-title="",HBTV (406p) [Not 24/7]
-http://hoabinhtvlive.746b3ddb.cdnviet.com/hoabinhtv/playlist.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:30]: Working link

File: channels/vn.m3u [hunk 2]
```diff
@@ -1,130 +1,1075 @@
...
+https://vips-livecdn.fptplay.net/sdc/quangnam_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="QuangNam.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Quảng Nam (576p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/sdc/quangnam_hls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="QuangNam.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Quảng Nam (404p)
 http://113.161.6.157:8081/hls-live/livepkgr/_definst_/liveevent/livestream.m3u8
-#EXTINF:-1 tvg-id="QuocHoi.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://www.quochoitv.vn/images/logo-qhtv.png" group-title="",Quốc Hội (720p)
-http://113.164.225.140:1935/live/quochoitvlive.stream_720p/playlist.m3u8
-#EXTINF:-1 tvg-id="SupremeMasterTV.vn" tvg-country="VN" tvg-language="Vietnamese;English" tvg-logo="" group-title="",Supreme Master TV (720p)
-https://lbs-us1.suprememastertv.com/720p.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:106]: Working link

File: channels/vn.m3u [hunk 3]
```diff
@@ -1,130 +1,1075 @@
...
-#EXTINF:-1 tvg-id="UniquelyThai.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Uniquely Thai (720p)
-https://livefta.malimarcdn.com/ftaedge00/uniquely.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="VajtswvTxojlus.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vajtswv Txojlus (720p)
-https://livefta.malimarcdn.com/ftaedge00/vajtswv.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="VanphenhTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vanphenh TV (720p)
-https://livefta.malimarcdn.com/ftaedge00/vanphenhtv.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="VatiLaoTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vati Lao TV (720p)
-https://livefta.malimarcdn.com/ftaedge00/vatilaotv.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="VietMyTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://s30.postimg.org/s0aco3hzl/VIET_MY.png" group-title="",Việt Mỹ TV (480p) [Not 24/7]
-http://68.235.37.11:1935/vietmagazine/vietmagazine/playlist.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:122]: Working link

File: channels/vn.m3u [hunk 4]
```diff
@@ -1,130 +1,1075 @@
...
-#EXTINF:-1 tvg-id="VinhLong1.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vĩnh Long 1 [Offline]
-http://htv-drm-live-cdn.fptplay.net/CDN-FPT02/THVL1-HD-1080p/playlist.m3u8
-#EXTINF:-1 tvg-id="VinhLong2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vĩnh Long 2 [Offline]
-http://htv-drm-live-cdn.fptplay.net/CDN-FPT02/THVL2-HD-1080p/playlist.m3u8
-#EXTINF:-1 tvg-id="VITV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://tduc.tk/logo/vitv.png" group-title="",VITV (180p)
-http://210.86.230.202:8134/hls-live/livepkgr/_definst_/liveevent/livestream.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:128]: Working link

File: channels/vn.m3u (hunk 1 above)
Comment (taodicakhia) [reply channels/vn.m3u:30]: #EXTINF:-1 tvg-id="GoodIdeaTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Good Idea TV (720p)
https://livefta.malimarcdn.com/ftaedge00/gooditv.sdp/playlist.m3u8
In #5293 

#EXTINF:-1 tvg-id="HBTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://i.imgur.com/BnFWgS7.png" group-title="",HBTV (406p) [Not 24/7]
http://hoabinhtvlive.746b3ddb.cdnviet.com/hoabinhtv/playlist.m3u8
Exitsted

File: channels/vn.m3u (hunk 2 above)
Comment (taodicakhia) [reply channels/vn.m3u:106]: #EXTINF:-1 tvg-id="QuocHoi.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://www.quochoitv.vn/images/logo-qhtv.png" group-title="",Quốc Hội (720p)
http://113.164.225.140:1935/live/quochoitvlive.stream_720p/playlist.m3u8
Existed

File: channels/vn.m3u (hunk 3 above)
Comment (taodicakhia) [reply channels/vn.m3u:122]: #EXTINF:-1 tvg-id="VatiLaoTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vati Lao TV (720p)
https://livefta.malimarcdn.com/ftaedge00/vatilaotv.stream/playlist.m3u8
In #5293 
#EXTINF:-1 tvg-id="VietMyTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://s30.postimg.org/s0aco3hzl/VIET_MY.png" group-title="",Việt Mỹ TV (480p) [Not 24/7]
http://68.235.37.11:1935/vietmagazine/vietmagazine/playlist.m3u8
Not in Vietnam

File: channels/vn.m3u (hunk 4 above)
Comment (taodicakhia) [reply channels/vn.m3u:128]: #EXTINF:-1 tvg-id="VinhLong2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="" group-title="",Vĩnh Long 2 [Offline]
http://htv-drm-live-cdn.fptplay.net/CDN-FPT02/THVL2-HD-1080p/playlist.m3u8
Not working
image

Comment (taodicakhia) [reply channels/vn.m3u:128]: #EXTINF:-1 tvg-id="VITV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://tduc.tk/logo/vitv.png" group-title="",VITV (180p)
http://210.86.230.202:8134/hls-live/livepkgr/_definst_/liveevent/livestream.m3u8

Update later

File: channels/vn.m3u (hunk 2 above)
Comment (taodicakhia) [reply channels/vn.m3u:106]: #EXTINF:-1 tvg-id="SupremeMasterTV.vn" tvg-country="VN" tvg-language="Vietnamese;English" tvg-logo="" group-title="",Supreme Master TV (720p)
https://lbs-us1.suprememastertv.com/720p.m3u8

Not in Vietnam

File: channels/vn.m3u (hunk 1 above)
Comment (freearhey) [reply channels/vn.m3u:30]: > Exitsted

?

File: channels/vn.m3u (hunk 2 above)
Comment (freearhey) [reply channels/vn.m3u:106]: Where?

File: channels/vn.m3u (hunk 3 above)
Comment (freearhey) [reply channels/vn.m3u:122]: >#EXTINF:-1 tvg-id="VietMyTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://s30.postimg.org/s0aco3hzl/VIET_MY.png" group-title="",Việt Mỹ TV (480p) [Not 24/7]
http://68.235.37.11:1935/vietmagazine/vietmagazine/playlist.m3u8
Not in Vietnam

Where is it?

File: channels/vn.m3u (hunk 4 above)
Comment (freearhey) [reply channels/vn.m3u:128]: As I said before, all changes must be included in the pull request in order for us to approve it.

File: channels/vn.m3u (hunk 1 above)
Comment (taodicakhia) [reply channels/vn.m3u:30]: > > Exitsted
> 
> ?

#EXTINF:-1 tvg-id="HoaBinh.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",Hòa Bình (406p)
http://hoabinhtvlive.746b3ddb.cdnviet.com/hoabinhtv/chunklist.m3u8

File: channels/vn.m3u (hunk 2 above)
Comment (taodicakhia) [reply channels/vn.m3u:106]: > #EXTINF:-1 tvg-id="SupremeMasterTV.vn" tvg-country="VN" tvg-language="Vietnamese;English" tvg-logo="" group-title="",Supreme Master TV (720p) https://lbs-us1.suprememastertv.com/720p.m3u8
> 
> Not in Vietnam

It's in Taiwan (International Channel)

File: channels/vn.m3u (hunk 3 above)
Comment (taodicakhia) [reply channels/vn.m3u:122]: > > #EXTINF:-1 tvg-id="VietMyTV.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="https://s30.postimg.org/s0aco3hzl/VIET_MY.png" group-title="",Việt Mỹ TV (480p) [Not 24/7]
> > http://68.235.37.11:1935/vietmagazine/vietmagazine/playlist.m3u8
> > Not in Vietnam
> 
> Where is it?

US

File: channels/vn.m3u [hunk 5]
```diff
@@ -1,130 +1,1075 @@
...
-https://livefta.malimarcdn.com/ftaedge00/briantv.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="BRT.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="http://brt.vn/common/v2/images/Banner150219.png" group-title="",BRT (360p)
+#EXTINF:-1 tvg-id="VTV1.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV1 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/hda1/vtv1hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV1.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV1 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/hda1/vtv1hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV1.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV1 (1080p)
+https://drfamaga5qliv.vcdn.cloud/vtv01/index.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:9]: Links no longer work.

File: channels/vn.m3u [hunk 6]
```diff
@@ -1,130 +1,1075 @@
...
+#EXTINF:-1 tvg-id="VTV1.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV1 (720p)
+https://mutixx5hq1liv.akamaized.net/vtv1/index.m3u8
+#EXTINF:-1 tvg-id="VTV2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV2 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/hda1/vtv2_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV2 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/hda1/vtv2_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV2 (1080p)
+https://drfamaga5qliv.vcdn.cloud/vtv02/index.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:19]: Links no longer work.

File: channels/vn.m3u [hunk 7]
```diff
@@ -1,130 +1,1075 @@
...
+#EXTINF:-1 tvg-id="VTV2.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV2 (720p)
+https://mutixx5hq1liv.akamaized.net/vtv2/index.m3u8
+#EXTINF:-1 tvg-id="VTV3.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV3 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/hda1/vtv3hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV3.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV3 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/hda1/vtv3hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV3.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV3 (1080p)
+https://drfamaga5qliv.vcdn.cloud/vtv03/index.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:29]: Links no longer work.

File: channels/vn.m3u [hunk 8]
```diff
@@ -1,130 +1,1075 @@
...
+#EXTINF:-1 tvg-id="VTV3.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV3 (720p)
+https://mutixx5hq1liv.akamaized.net/vtv3/index.m3u8
+#EXTINF:-1 tvg-id="VTV4.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV4 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/hda1/vtv4_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV4.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV4 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/hda1/vtv4_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV4.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV4 (1080p)
+https://drfamaga5qliv.vcdn.cloud/vtv04/index.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:39]: Links no longer work.

File: channels/vn.m3u [hunk 9]
```diff
@@ -1,130 +1,1075 @@
...
+#EXTINF:-1 tvg-id="VTV4.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV4 (720p)
+https://mutixx5hq1liv.akamaized.net/vtv4/index.m3u8
+#EXTINF:-1 tvg-id="VTV5.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV5 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/hda2/vtv5hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV5.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV5 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/hda2/vtv5hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV5.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV5 (1080p)
+https://drfamaga5qliv.vcdn.cloud/vtv05/index.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:49]: Links no longer work.

File: channels/vn.m3u [hunk 10]
```diff
@@ -1,130 +1,1075 @@
...
+#EXTINF:-1 tvg-id="VTV5.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV5 (720p)
+https://mutixx5hq1liv.akamaized.net/vtv5/index.m3u8
+#EXTINF:-1 tvg-id="VTV6.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV6 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/hda1/vtv6hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV6.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV6 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/hda1/vtv6hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV6.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV6 (1080p)
+https://drfamaga5qliv.vcdn.cloud/vtv06/index.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:59]: Links no longer work.

File: channels/vn.m3u [hunk 11]
```diff
@@ -1,130 +1,1075 @@
...
+#EXTINF:-1 tvg-id="VTV6.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV6 (720p)
+https://mutixx5hq1liv.akamaized.net/vtv6/index.m3u8
+#EXTINF:-1 tvg-id="VTV7.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV7 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/hda3/vtv7hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV7.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV7 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/hda3/vtv7hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV7.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV7 (1080p)
+https://drfamaga5qliv.vcdn.cloud/vtv07/index.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:69]: Links no longer work.

File: channels/vn.m3u [hunk 12]
```diff
@@ -1,130 +1,1075 @@
...
+#EXTINF:-1 tvg-id="VTV7.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV7 (720p)
+https://mutixx5hq1liv.akamaized.net/vtv7/index.m3u8
+#EXTINF:-1 tvg-id="VTV8.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV8 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/hda2/vtv8hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV8.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV8 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/hda2/vtv8hd_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV8.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV8 (1080p)
+https://drfamaga5qliv.vcdn.cloud/vtv08/index.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:79]: Links no longer work.

File: channels/vn.m3u [hunk 13]
```diff
@@ -1,130 +1,1075 @@
...
+#EXTINF:-1 tvg-id="VTV9.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV9 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://vips-livecdn.fptplay.net/hda2/vtv9_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV9.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV9 (1080p)
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:92.0) Gecko/20100101 Firefox/92.0
+https://livecdn.fptplay.net/hda2/vtv9_vhls.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="VTV9.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV9 (1080p)
+https://drfamaga5qliv.vcdn.cloud/vtv09/index.m3u8
+#EXTINF:-1 tvg-id="VTV9.vn" tvg-country="VN" tvg-language="Vietnamese" tvg-logo="",VTV9 (1080p) [Geo-blocked]
+https://htv-drm-live-cdn.fptplay.net/CDN-FPT02/VTV9-HD-1080p/playlist.m3u8
```
Comment (freearhey) [inline channels/vn.m3u:91]: Links no longer work.

**Review by freearhey — CHANGES_REQUESTED** [review]
Links from fptplay.net and vcdn.cloud no longer works. Please check them again and remove broken one.

File: channels/vn.m3u (hunk 5 above)
Comment (taodicakhia) [reply channels/vn.m3u:9]: image
image

**Pull request closed without merging**